includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -94,6 +98,9 @@ public function __construct( WC_Payments_API_Client $payments_api_client, WC_Pay
 		if ( ! is_admin() ) {
 			add_filter( 'woocommerce_gateway_title', [ $this, 'maybe_filter_gateway_title' ], 10, 2 );
 		}
+
+		add_action( 'woocommerce_order_payment_status_changed', [ 'WCPay\Payment_Methods\UPE_Payment_Gateway', 'remove_upe_payment_intent_from_session' ], 10, 0 );
```

Comment:
```suggestion
		add_action( 'woocommerce_order_payment_status_changed', [ __CLASS__, 'remove_upe_payment_intent_from_session' ], 10, 0 );
```
I think you can save a few characters and safely use the `__CLASS__` here. Or at the least, I don't think you need the full absolute class name, since the class is namespaced.

Reply:
Fixed in c6ff6fa :)